### DISCUSSION TITLE
How am I expected to use the search results given??

### DISCUSSION THREAD

**Discussion by Robby-Sodhi** [tweepy/tweepy#1485] [Questions] [answered]
https://pastebin.com/wzA9uQfN

I get something like this. it isn't a dictionary or a list how am I expected to use the data within it?

**Comment by Robby-Sodhi** [comment]
solution:
https://www.reddit.com/r/learnpython/comments/kkyhn5/trying_to_use_tweepy_to_search_twitter_not/?utm_source=share&utm_medium=web2x&context=3

**Comment by Harmon758** [comment, marked as answer]
This is a list with a single Status object, which is how Tweepy models the [Tweet object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet) that Twitter's API returns as JSON.
You can access the JSON with `Status._json`. Otherwise, you can directly access the attributes of the Status/Tweet object.

**Reply by Robby-Sodhi** [reply]
Thank you using attributes worked for me
